### ISSUE TITLE
An issue with function "visualize_image_attr()"

### ISSUE THREAD

**Issue by Bubble-Sword** [pytorch/captum#1163]
## 🐛 Bug

I run the tutorial at "https://captum.ai/tutorials/Segmentation_Interpret" through Google Colab

When proceeding to In [12]:
<img width="765" alt="image" src="https://github.com/pytorch/captum/assets/39286423/614a622b-c201-4617-ab81-984b1967a5c5">

an error is reported
<img width="1072" alt="image" src="https://github.com/pytorch/captum/assets/39286423/bf7b59fd-6adb-4db9-b48d-84b985600e7f">

And then I check the source code of "visualization.py" on the git hub, the code at line 250 is:
<img width="682" alt="image" src="https://github.com/pytorch/captum/assets/39286423/9e4f65a3-aad6-4acc-944b-9ac214902a4d">

## To Reproduce

1. run the notebook download from "https://captum.ai/tutorials/Segmentation_Interpret"

## Expected behavior

the ndarray can be visualized

## Environment
Google Colab
Python 3.10.12
Pytorch 2.0.1+cu118

install with: !pip install captum

```

 - Captum / PyTorch Version (e.g., 1.0 / 0.4.0):
 - OS (e.g., Linux):
 - How you installed Captum / PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->

**Comment by kainat-altaf-cau** [comment]
I am facing the same issue.

**Comment by vivekmig** [comment]
Hi @Bubble-Sword and @kainat-altaf-cau , we have fixed this issue in #1118 , but this fix is not yet in a release package. In the meantime, you can install Captum from source (master) as follows to avoid this issue:
```
pip install git+https://github.com/pytorch/captum.git
```
